allennlp/data/batch.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, instances: Iterable[Instance]) -> None:
         """
         super().__init__()
 
-        self.instances: List[Instance] = ensure_list(instances)
```

Comment:
When there are type annotations like this, it's generally because mypy complained about something.  Make sure that mypy doesn't complain if you remove them.